lib/date.js

```diff
@@ -5,18 +5,18 @@ export function exhibitionPeriod(startAt, endAt) {
   const endMoment = moment(endAt);
   const thisMoment = moment();
 
-  let startFormat = 'MMMM Do';
+  let startFormat = 'MMM D';
   if (startMoment.year() !== endMoment.year()) {
     startFormat = startFormat.concat(', YYYY');
   }
 
-  let endFormat = 'Do';
+  let endFormat = 'D';
   if (endMoment.year() !== thisMoment.year()) {
     endFormat = endFormat.concat(', YYYY');
   }
   if (!(startMoment.year() === endMoment.year() && startMoment.month() === endMoment.month())) {
```

Comment:
@dzucconi I didn’t end up using those moment.js methods. I had hoped it could have simplified this conditional, but it really didn’t do much, so instead I opted for plain operators, which everyone should be able to understand.